matlab/plot_queue_sample_path.m

```diff
@@ -0,0 +1,13 @@
+function plot_queue_sample_path(input, N_client)
+
+[m,n] = size(input);
+T = m/N_client;
```

Comment:
There might be a caveat that when channel is bad, and all poll_num replies fail, and there is missing data for some intervals.